<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInForm</class>
 <widget class="QWidget" name="LogInForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/mainapp/sources/appico.ico</normaloff>:/mainapp/sources/appico.ico</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="signIn">
    <attribute name="title">
     <string>Вход</string>
    </attribute>
    <widget class="QLineEdit" name="loginField">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>400</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_login">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Логин&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_password">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Пароль&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordField">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>400</width>
       <height>24</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="signInButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="signUp">
    <attribute name="title">
     <string>Зарегистрироваться</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
